client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/pages/participant-detail-manage/components/participant-create-wizard/participant-create-wizard.component.ts

```diff
@@ -319,4 +321,32 @@ export class ParticipantCreateWizardComponent extends BaseMeetingComponent imple
             }
         }
     }
+
+    public get isLockedOutAndCanManage(): boolean {
+        const lockedOutHelper = this.personalInfoFormValue?.locked_out ?? this.user?.is_locked_out;
+        return lockedOutHelper && this.checkSelectedGroupsCanManage;
+    }
+
+    public get lockoutCheckboxDisabled(): boolean {
+        const notChanged = (this.personalInfoFormValue?.locked_out ?? null) === null;
+        const isLockedOut = this.user?.is_locked_out;
+        return notChanged && !isLockedOut && this.checkSelectedGroupsCanManage;
+    }
+
+    public updateLockout(): void {
+        const check = this.detailView.personalInfoForm.get(`locked_out`).disabled !== this.lockoutCheckboxDisabled;
+        if (check) {
```

Comment:
Just do the check directly or name the variable reasonable.

Reply:
cleaned.